app/utils/socket.js

```diff
@@ -37,6 +37,11 @@ export default Ember.Object.extend(Ember.Evented, {
     this.set('metadata', metadata||this.get('metadata')||{});
 
     var url = this.get('url');
+
```

Comment:
This doesn't seem like the right fix.. if the URL is null from the stats response it should just not even try to create a socket.